deployment/terraform/templates/default_configuration.nix

```diff
@@ -5,4 +5,16 @@
   services.fail2ban.enable = true;
   users.extraUsers.root.openssh.authorizedKeys.keys = [ ${root_ssh_keys} ];
 
+  # we need to configure the binary caches here otherwise the
+  # initial morph deployment cannot substitute anything causing
+  # everything to be uploaded from the deployer machine
+  nix = {
```

Comment:
Can we make this a module that we import? I can see at least three other uses.

Reply:
fair point - i'll take a look! Might be that the other places should indeed just import `default_configuration.nix` ..

Reply:
So while I agree that more code reuse/modularization ought to be taking place here i can't do that:

The file in question is a _terraform template file_ so it'll just be inserted as-is and I can't import any other (local) files.

Reply:
Huh. It imports a module on line 3, but I guess it does that from `NIX_PATH` which somehow works...